### PR TITLE
data: joeboadiDev

### PR DESCRIPTION
**Pull request by joeboadiDev**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6138"></a>

### CODE REVIEW

File: data/joeboadiDev.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name" : "Joe Boadi,
```
Comment (priyankarpal) [inline data/joeboadiDev.json:2]: Add `"` at the end. 
```suggestion
  "name" : "Joe Boadi",
```

File: data/joeboadiDev.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "displayStatsPublic" : "true",
```
Comment (priyankarpal) [inline data/joeboadiDev.json:3]: remove the line currently it's not needed.
```suggestion

```

File: data/joeboadiDev.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "type" : "personal",
+  "bio" : "Ex-allied health, Tech enthusiast, Software Engineer",
+  "avatar" : "https://github.com/joeboadiDev.png",
```
Comment (priyankarpal) [inline data/joeboadiDev.json:6]: same here also remove the line.
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by joeboadiDev — COMMENTED** [review]
excess lines and punctuations corrected.

File: data/joeboadiDev.json (hunk 2 above)
Comment (joeboadiDev) [reply data/joeboadiDev.json]: Done.

File: data/joeboadiDev.json (hunk 3 above)
Comment (joeboadiDev) [reply data/joeboadiDev.json]: Done, thanks.

**Review by joeboadiDev — COMMENTED** [review]
REVIEW changes

File: data/joeboadiDev.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name" : "Joe Boadi",
+  "type" : "personal",
+  "bio" : "Ex-allied health, Tech enthusiast, Software Engineer",
+  "links" : [
+      {
+        "name" : "Twitter",
+        "url" : "https://twitter.com/joeboaadiDev_",
```
Comment (priyankarpal) [inline data/joeboadiDev.json:8]: correct your twitter profile link

```suggestion
        "url" : "https://twitter.com/joeboadiDev_",
```

File: data/joeboadiDev.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        "icon" : "FaTwitter"
+      },
+       {
+        "name" : "Github",
+        "url" : "https://github.com/joeboaadiDev",
```
Comment (priyankarpal) [inline data/joeboadiDev.json:13]: same here also fix your github profile link

```suggestion
        "url" : "https://github.com/joeboadiDev",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by SandeepKrSuman — DISMISSED** [review]
Thanks, @joeboadiDev 🎉
Looks good to me!

**Review by harshsingh32 — DISMISSED** [review]

**Review by joeboadiDev — COMMENTED** [review]
Change made.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**